sshmuxer/requests.go

```diff
@@ -171,6 +177,10 @@ func handleRemoteForward(newRequest *ssh.Request, sshConn *utils.SSHConnection,
 			return
 		}
 
+		//Reply that the session is successfully set up with the port that was assigned (which may be different than what was requested)
```

Comment:
Taking a second glance at this, we actually might want to reply true regardless of the scenario. We don't have to include the port for each type of forward (http or alias), but I also don't think it would hurt anything